vectortile/data/vector_tile.proto

```diff
@@ -0,0 +1,78 @@
+package vector_tile;
+
+option optimize_for = LITE_RUNTIME;
+
+message Tile {
```

Comment:
@jpolchlo the ways to change names for the generated items are:

1. Change the schema
2. Add type aliases